docs/installation.rst

```diff
@@ -78,7 +56,26 @@ this method by following the steps described below.
       .. note::
 
         The complete developer guidelines can be found :ref:`here <developer_guidelines>`.
+        
+    b. Non-developers can install from specific releases using pip-
+
+      ::
+
+        $ pip install git+https://github.com/tardis-sn/tardis.git@{tag}
+
+      For example- 
+
+      ::
+      
+        $ pip install git+https://github.com/tardis-sn/tardis.git@release-{yyyy.mm.dd} 
+        
+        Please fill in the year, month, and date of the released version you want to use.
+
+      or trying the most recent, unreleased changes from upstream.
 
+      ::
+
+        $ pip install git+https://github.com/tardis-sn/tardis.git@master
 
 4. Once finished working, you can deactivate your environment.
 
```

Comment:
Please add "$ wget -q https://github.com/tardis-sn/tardis/releases/latest/download/conda-{platform}-64.lock" in line 110 with the same indentation with the line below that "$ conda update xxx"